src/modules/Dashboard/containers/ImagesTabMenuContainer.ts

```diff
@@ -1,11 +1,11 @@
 import {connect} from 'react-redux';
 
 import {
-		getAllImagesCount,
-		getBuildingImagesCount,
-		getFailedImagesCount,
-		getSuccededImagesCount,
-		} from 'common/selectors/entities';
+	getAllImagesCount,
+	getBuildingImagesCount,
+	getFailedImagesCount,
+	getSuccededImagesCount,
+} from 'common/selectors/entities';
```

Comment:
why are these changes, made by Alena, is in your branch?

Reply:
Because why do we need to create separate task just for indention fix, when i can notice it and correct by pull request?
I thought we don't need to overtask our trello